zyte_common_items/components.py

```diff
@@ -285,3 +285,42 @@ class RealEstateArea(Item):
 
     #: Area in the raw format, as it appears on the website.
     raw: str
+
+
+@attrs.define(kw_only=True)
+class Header(Item):
+    """A http header value"""
+
+    #: Name of header
+    name: str
+
+    #: Value of the header
+    value: str
+
+
+@attrs.define(kw_only=True)
+class Request(Item):
+    """Describe a HTTP request to load a page"""
```

Comment:
```suggestion
    """Describe a web request to load a page"""
```

Probably safer to stay away from HTTP wording on this one.